### PR TITLE
1up60hte hhkb layout support

### PR DESCRIPTION
**Pull request by bubnick**
## Description

Support for HHKB-style layout for 1upkeyboards HTE PCB

As well as minor keymap fixes for the default layout for the same board.

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/1upkeyboards/1up60hte/1up60hte.h [hunk 1]
```diff
@@ -33,4 +33,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 	{ K400,  K401,  K402,  KC_NO, KC_NO, KC_NO, K406,  KC_NO, KC_NO, KC_NO, K410,  K411,  K412,  K413 }  \
 }
 
+/* HHKB Variant */
+#define LAYOUT_hhkb( \
```
Comment (mechmerlin) [inline keyboards/1upkeyboards/1up60hte/1up60hte.h]: you can actually take out the `KC_NO` and use `LAYOUT_60_hhkb` to use our default layouts in https://github.com/qmk/qmk_firmware/tree/master/layouts/default

**Comment by mechmerlin** [comment]
Also, I noticed this board doesn't have a `readme.md` would you mind putting one in using a template like the ones in the other 1upkeyboards keyboards?

File: keyboards/1upkeyboards/1up60hte/1up60hte.h (hunk 1 above)
Comment (bubnick) [reply keyboards/1upkeyboards/1up60hte/1up60hte.h]: I wasn't entirely clear on what you meant by this, but I changed to the following:
`/* HHKB Variant */
#define LAYOUT_60_hhkb( \
  K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K012, K013, \
	K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K112, K113, \
	K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K212,       \
	K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K312,       \
	      K401, K402,                  K406,                    K410, K411, K412, K413  \
)`
Is this correct?

Comment (mechmerlin) [reply keyboards/1upkeyboards/1up60hte/1up60hte.h]: QMK has a bunch of default layouts such as https://github.com/qmk/qmk_firmware/tree/master/layouts/default 

This gives a good standard KLE file for `info.json` work. However the other benefit is that if you put `LAYOUTS = 60_hhkb` in your `rules.mk` file, anyone who has a community keymap that targets a LAYOUT_60_hhkb can use their keymap.

Comment (bubnick) [reply keyboards/1upkeyboards/1up60hte/1up60hte.h]: I made the changes necessary + did a make to ensure it was properly compiling.
Commit: 1648a62

**Pull request merged**